### PR TITLE
added responsive images component

### PR DESCRIPTION
**Pull request by shivanigangadharan**
Added responsive images. 

Unable to add video without lag, so added cropped screenshots as follows - 

**Large screen :-** 
image

**Medium screen :-** 
Screenshot (690)

**Small screen :-** 
Screenshot (691)

### CODE REVIEW

File: pages/docs/docs.html [hunk 1]
```diff
@@ -139,13 +142,31 @@
...
         </div>
-        <!-- <div class="component">
-                <div class="container-snackbar">
-                    <span class="text"> Can't send photo. Retry in 5 seconds. </span>
-                    <span class="retry"> RETRY </span>
-                    <span class="x"> x </span>
+
+        <div class="page-content" id="image">
+            <div>
+                <text class="title"> Image</text>
```
Comment (junaidshaikh-js) [inline pages/docs/docs.html]: Which element is this `<text>`?

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work! 

I couldn't understand what the <text> element does, never heard about it.

**Review by Maianki — APPROVED** [review]
Looks good to merge.

File: pages/docs/docs.html (hunk 1 above)
Comment (shivanigangadharan) [reply pages/docs/docs.html]: > Which element is this `<text>`?

My bad, just wanted to give it a class. Now switched it to `<span>` in commit [6c3fe59](https://github.com/shivanigangadharan/style-ui/pull/12/commits/6c3fe595bacc542c6d036df2ec2fe446e4c937ea) 👍

**Pull request merged**